.github/workflows/deploy-mainnet-vaults-keeper-helper.yaml

```
@@ -0,0 +1,40 @@
+      url: https://term-finance.github.io/yearn-v3-term-vault/
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: forge install
+      - run: forge build
+      - run: forge tree
+      - name: Run deploy script
+        run: forge script script/TermVaultsKeeper.s.sol:TermVaultsKeeperDeployer --rpc-url $RPC_URL --verify --vvvv --broadcast
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:79: Double quote to prevent globbing and word splitting

(shellcheck)